src/client/pages/DSM.js

```diff
@@ -1,41 +1,46 @@
-import React from 'react';
```

Comment:
Since we are transforming to function component style, could you switch back to function component here? Thanks.

Reply:
Removed the file, the old version was committed, the current version is already in master